addons/live-comments/addon.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Live comment refreshing",
+  "description": "Refreshes comments on studios and projects to create a near-live view.",
```

Comment:
I think it'd be better to just say "Automatically refreshes website comments to create a near-live view." And specify that it doesn't work on user profiles in a `warning`

Reply:
i forgot those exist